src/asciidoc/reference/deprecated/withkojiVCC.adoc

```diff
@@ -474,14 +437,14 @@ Enables distinct template experiences for different viewing contexts.
 
 * `about` – Manages the context for the remix landing page (About page).
 To use this context:
-** *Define an About page experience* – Enable the `AboutContext` <<_feedsdk, entitlement>>, and define a view intended for the About page.
+** *Define an About page experience* – Enable the `AboutContext` entitlement, and define a view intended for the About page.
```

Comment:
Are these and below still supposed to be here?

Reply:
Nah, we don't need these deprecated instructions. Removed and just left the generic bullet (for now, until we remove those packages from the site entirely).